### ISSUE TITLE
Lock button for augmentations

### ISSUE THREAD

**Issue by fwild** [WEKIT-ECS/MIRAGE-XR#1484]
Add a lock button for augmentations, to prevent the user from accidentially moving them

**Comment by fominykh** [comment]
From the list of augmentation, we should understand if the augmentation is locked or not.
Locking / unlocking can be done in the three dots. Some augmentations are also unmovable.

**Comment by robhillman97** [comment]
I have been working on this and I have it set up currently so that the lock disables the object manipulator and bounding box components of an augmentation. This means that you cannot move or re-size the object directly, however, if you move the task station the augmentation will still move relative to it. See below:

![AugmentationLock2](https://github.com/WEKIT-ECS/MIRAGE-XR/assets/65968011/559db5b8-150e-4e1b-b646-06b397e66f4a)

I think this is how we want this to work but wanted to check if the lock should cause the augmentation to ignore the task station moving too. @fwild @fominykh what do you think?

**Comment by fominykh** [comment]
agreed
